From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Tue, 27 Aug 2019 11:00:34 -0700
Subject: [PATCH] Don't emit compiler version requirement.

We're not getting the proper provides, and it's not needed for
a distro like clearlinux where we rebuild dependent packages
anyway.
---
 fileattrs/ocaml.attr | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fileattrs/ocaml.attr b/fileattrs/ocaml.attr
index 5fd0870..8246c8d 100644
--- a/fileattrs/ocaml.attr
+++ b/fileattrs/ocaml.attr
@@ -1,3 +1,3 @@
 %__ocaml_provides	%{_rpmconfigdir}/ocaml-find-provides.sh
-%__ocaml_requires	%{_rpmconfigdir}/ocaml-find-requires.sh
+%__ocaml_requires	%{_rpmconfigdir}/ocaml-find-requires.sh -c
 %__ocaml_magic		^(Objective caml|OCaml) .*$
